library/src/skins/default/src/sass/modules/tool/portal/_portal.scss

```diff
@@ -293,80 +283,105 @@ a[href]:not(.btn):hover {
+          }
+        }
+        &:hover {
+          .bi-pin {
+            color: var(--sakai-selected-page-color);
+          }
+          a, button {
+            color: var(--sakai-selected-page-color);
+          }
+        }
+      }
+    }
   }
 
   .btn-nav,
   .btn-site-opt {
-    @include btn-base();
-    --button-color: (--site-nav-btn-color);
-    --button-bg: var(--site-nav-btn-bg);
-    --button-border-color: var(--site-nav-btn-border-color);
-    --button-hover-color: var(--site-nav-btn-hover-color);
-    --button-hover-bg: var(--site-nav-btn-hover-bg);
-    --button-hover-border-color: var(--site-nav-btn-hover-border-color);
+    @include button-variant($white, $black);
+    --bs-btn-color: var(--site-nav-btn-color);
+    --bs-btn-bg: var(--site-nav-btn-bg);
+    --bs-btn-border-color: var(--site-nav-btn-border-color);
+    --bs-btn-hover-color: var(--site-nav-btn-hover-color);
+    --bs-btn-hover-bg: var(--site-nav-btn-hover-bg);
+    --bs-btn-hover-border-color: var(--site-nav-btn-hover-border-color);
+    --bs-btn-focus-shadow-rgb: 249, 250, 251;
+    --bs-btn-active-color: var(--site-nav-btn-active-color);
+    --bs-btn-active-bg: var(--site-nav-btn-active-bg);
+    --bs-btn-active-border-color: var(--site-nav-btn-active-border-color);
+    --bs-btn-disabled-color: var(--site-nav-btn-disabled-color);
+    --bs-btn-disabled-bg: var(--site-nav-btn-disabled-bg);
+    --bs-btn-disabled-border-color: var(--site-nav-btn-disabled-border-color);
     --bs-btn-focus-shadow-rgb: 38, 88, 129;
-    --button-active-color: var(--site-nav-btn-active-color);
-    --button-active-bg: var(--site-nav-btn-active-bg);
-    --button-active-border-color: var(--site-nav-btn-active-border-color);
-    --button-disabled-color: var(--site-nav-btn-disabled-color);
-    --button-disabled-bg: var(--site-nav-btn-disabled-bg);
-    --button-disabled-border-color: var(--site-nav-btn-disabled-border-color);
-    .is-current-site .site-list-item-head > & {
-      --button-color: (--site-nav-btn-current-color);
-      --button-bg: var(--site-nav-btn-current-bg);
-      --button-border-color: var(--site-nav-btn-current-border-color);
-      --button-hover-color: var(--site-nav-btn-current-hover-color);
-      --button-hover-bg: var(--site-nav-btn-current-hover-bg);
-      --button-hover-border-color: var(--site-nav-btn-current-hover-border-color);
-      --bs-btn-focus-shadow-rgb: 38, 88, 129;
-      --button-active-color: var(--site-nav-btn-current-active-color);
-      --button-active-bg: var(--site-nav-btn-current-active-bg);
-      --button-active-border-color: var(--site-nav-btn-current-active-border-color);
-      --button-disabled-color: var(--site-nav-btn-current-disabled-color);
-      --button-disabled-bg: var(--site-nav-btn-current-disabled-bg);
-      --button-disabled-border-color: var(--site-nav-btn-current-disabled-border-color);
+    &:hover {
+      --bs-btn-color: var(--site-nav-btn-active-color);
```

Comment:
do you think it will have some impact where it would be hard to test right now?

Reply:
What do you mean? I think all these css var changes are scoped to be inside the site nav stuff, and I've tested site nav a lot. Do you think these changes will affect things outside of the site nav? That would be bad.

Reply:
I think you are right :+1: